chall02/isoria-g.py

```diff
@@ -0,0 +1,12 @@
+import sys
+letter=("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
+morse=(".-","-...","-.-.","-..",".","..-.","--.","....","..",".---","-.-",".-..","--","-.","---",".--.","--.-",".-.","...","-","..-","...-",".--","-..-","-.--","--..")
+if len(sys.argv) == 2 and (all(c.isalpha() for c in sys.argv[1])):
+    string=(sys.argv[1]).lower()
+    i = 0
+    while i <= 25:
+        string = string.replace(letter[i], morse[i])
+        i=i+1
```

Comment:
Using a for here would have been a more pythonic solution
```suggestion
    for i, code in enumerate(morse):
        string = string.replace(letter[i], code)
```
A further amelioration would be a comprehension using a dictionary as mentioned earlier
```suggestion
string = "".join(morse[c] for c in string)
```